docs/content/en/docs/reference/artifacts.md

```diff
@@ -266,21 +270,21 @@ These steps use `image-builder` to create an Ubuntu-based or RHEL-based image fo
 
       * `--os`: `ubuntu`
       * `--hypervisor`: For vSphere use `vsphere`
-      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, and 1-24.
+      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25.
```

Comment:
Our version support page (https://anywhere.eks.amazonaws.com/docs/reference/support/support-versions/) shows 1-21 at end of support on February 15, 2023. So I'm thinking we should either remove it here or change it there if it's still supported.

Reply:
Will extend this to 0.15.0 release date since that's when we plan to deprecate it officially.